app/models/token_and_user.rb

```diff
@@ -7,9 +7,11 @@ def assignments
     []
   end
 
-  def attributes
-    super.merge(status: "success",
-                session_ticket: @access_token.attributes[:token],
-                assignments: assignments)
+  def status
+    "success"
+  end
+
+  def session_ticket
+    @access_token.attributes[:token]
```

Comment:
1. I didn't catch where these new methods were called?
2. I'm guessing `assignemnts` are no longer needed?

Reply:
So it seems that active_model_serializers behaves differently with the upgraded version. It wasn't looking at the `attributes` method, so I had to manually define each of the attributes as methods (which AMS does use). `assignments` already existed above this code if you look at the full file.